.evergreen/config.yml

```diff
@@ -377,6 +377,32 @@ functions:
             MONGODB_API_VERSION=${MONGODB_API_VERSION} \
             .evergreen/run-tests.sh
 
+  "run serverless tests":
+    - command: shell.exec
+      type: test
+      params:
+        shell: bash
+        working_dir: "src"
+        script: |
+          # Exported before PREPARE_SHELL to avoid echo with xtrace
```

Comment:
I think it might be worth just disabling xtrace for these tests in case one of the later scripts accidentally echos these. We can still run `${PREPARE_SHELL}` but just disable it afterwards; I believe that's what we do in Swift.

Reply:
Done.